config/application.rb

```diff
@@ -20,6 +20,6 @@ class Application < Rails::Application
     # config.eager_load_paths << Rails.root.join("extras")
 
     config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
-
+    config.exceptions_app = self.routes
```

Comment:
Why is it in two places? Here and in the dev environment? And what exactly does that mean?

Reply:
Unnecessary leftover.